roles.json

```
@@ -371,26 +371,24 @@
         }
     },
     {
-        "role": "Core package release coordinator",
-        "url": "Core_package_release_coordinator",
+        "role": "Release team",
+        "url": "release_team",
         "people": [
+            "E. Madison Bray",
+            "Simon Conseil",
             "Thomas Robitaille",
-            "Brigitta Sip\u0151cz",
-            "Erik Tollerud"
+            "Brigitta Sip\u0151cz"
         ],
-        "role-head": "Core package release coordinator",
+        "role-head": "Release team",
         "responsibilities": {
-            "description": "Oversee the release process for the astropy core package, including:",
+            "description": "Oversee the release process for packages in the project, including:",
             "details": [
-                "Performing new releases",
-                "Coordinate releases of astropy-helpers with the astropy-helpers maintainer(s)",
-                "Maintaining the change log and \u201cwhat\u2019s new\u201d documentation page",
-                "Maintaining the bugfix branches",
-                "Updating Astropy\u2019s PyPI entry as needed",
+                "Carrying out releases of the core astropy package",
+                "Notifying the Distribution Coordinators of any core astropy package release",
                 "Working with the Community Engagement Coordinator to make release announcements via channels such as mailing lists and social media",
-                "Notifying the Distribution Coordinators of any release",
-                "Make sure that the package gets updated in conda and coordinate with distribution coordinators",
-                "Keep documentation for release process up to date"
+                "Keeping documentation for the release process of the core package up to date",
+                "Designing policies to improve the uniformity of release procedures for coordinated and infrastructure packages",
```

```suggestion
                "Designing policies to improve the uniformity of release procedures for the Astropy Project, particularly coordinated and infrastructure packages",
```

---

👎  on `particularly` as explicit is better than implicit. Adding the the Astropy Project to the end of the sentence would be good. `"for the coordinated and infrastructure packages of the Astropy Project" - so we are not taking up the maintenance of pieces of infra outside the project.

Of course, the team can be consulted for or asked to help out with affiliated packages, but this page describes the remit and responsibilities of this role. The project doesn't have mandate to with non coordinated packages, let alone design release procedure for downstream packages, if some of those follow ours, it's all good and nice, but given that there are ~50 affiliated packages, those should not fall inside our talks at all.

---

Ok, so is this what you're thinking @bsipocz ? I'm ok with that.
```suggestion
                "Designing policies to improve the uniformity of release procedures for the coordinated and infrastructure packages of the Astropy Project",
```